JenkinsfileRT

```diff
@@ -40,18 +40,23 @@ bc.conda_packages = ['python=3.6',
                      'astropy',
                      'ci-watson']
 
-bc.build_cmds = ["${configure_cmd} --release-with-symbols",
-                  "./waf build",
-                  "./waf install",
-                  "calacs.e --version"]
+bc.build_cmds = ["${configure_cmd} --release-with-symbols -v",
+                 "./waf build -v",
+                 "./waf install -v",
+                 "calacs.e --version"]
 bc.test_cmds = ["pytest tests --basetemp=tests_output --junitxml results.xml --bigdata --slow -v"]
 bc.failedUnstableThresh = 1
 bc.failedFailureThresh = 6
 
 // Upload artifacts
 bc.test_configs = [data_config]
 
+// Also run on OSX
+bc_osx = utils.copy(bc)
+bc_osx.nodetype = "osx"
+bc_osx.env_vars += ['CC=clang']
```

Comment:
Unresolved: We probably want to upload the OSX outputs too, but how and to where?

Reply:
Deferring to future work.